db/migrate/20170327100309_lock_plugin_roles.rb

```diff
@@ -0,0 +1,15 @@
+class LockPluginRoles < ActiveRecord::Migration
+  def up
+    Foreman::Plugin.all.each do |plugin|
```

Comment:
If you add plugin later, roles will get [locked](https://github.com/theforeman/foreman/pull/4415/files#diff-d0644101ad428cdee084ab59f3e9a43cR245) when registering the plugin. Migration just renames the roles that were modified.